runbotics/runbotics-desktop/src/action/general/general.action-handler.ts

```diff
@@ -44,8 +45,9 @@ export default class GeneralActionHandler extends StatelessActionHandler {
     async startProcess(
         request: DesktopRunRequest<GeneralAction.START_PROCESS, StartProcessActionInput>
     ): Promise<StartProcessActionOutput> {
-        const response = await orchestratorAxios.get<IProcess>(
-            `/api/processes/${request.input.processId}`,
+        const tenantId = this.storageService.getValue('tenantId');
```

Comment:
I know that, we have such implementation, but I don't think this is a good approach of keeping such information like `tenantId`. Maybe we should think about changing it in the future

Reply:
Good point, if it's stored in local storage there's a risk someone can exploit that to access resources of another tenant